### DISCUSSION TITLE
Regarding Lammps trajectroy

### DISCUSSION THREAD

**Discussion by aashuph16221** [MDAnalysis/mdanalysis#4595] [General]
Hello,

I have encountered while attempting to use MDAnalysis for analyzing LAMMPS MD trajectories. I would greatly appreciate your assistance in resolving this matter.

I am trying to load a trajectory file in the LAMMPS dump format (format="lammpsdump"), along with a corresponding topology file (v2.data).

u = mda.Universe('v2.data', 'data.production2.lip.lammpstrj',
                 topology_format="data", format="lammpsdump", atom_style='id resid type x y z',guess_bonds=True)

However, I consistently encounter the following error:This error occurs when MDAnalysis attempts to read lines from the trajectory file, suggesting that the number of values in each line does not match the expected format.

ValueError: too many values to unpack (expected 5)

I would greatly appreciate any guidance or assistance you can provide in resolving this issue. I look forward to your response and assistance.

Regards,

Aashish

**Comment by hmacdope** [comment]
Can you post an example of your code + your DATA file + your lammpstrj file please?

**Reply by hmacdope** [reply]
I will mark this as closed soon if no files forthcoming.
